doc/changelog/07-vernac-commands-and-options/15528-master+adding-specific-scheme-boolean-equality.rst

```diff
@@ -0,0 +1,10 @@
+- **Added:**
+  New command :cmd:`Scheme Boolean Equality` to generate the Boolean
+  equality associated to an inductive type whose equality is
+  decidable; on the contrary of :cmd:`Scheme Equality`, this does not
+  try to prove the decidability of the type; :cmd:`Scheme Boolean
+  Equality` can be useful when Coq is able to generate the Boolean
+  equality but not powerful enough to prove the decidability of
+  equality
+  (`#15528 <https://github.com/coq/coq/pull/15528>`_,
```

Comment:
```suggestion
  (`#15526 <https://github.com/coq/coq/pull/15526>`_,
```

Reply:
Huh, sorry about it and thanks for telling!